test/metabase/models/field_values_test.clj

```diff
@@ -26,7 +26,7 @@
                                                 :visibility_type          :details-only
                                                 :base_type                :type/Text}))
 
-(expect false (field-should-have-field-values? {:special_type    nil
+(expect true (field-should-have-field-values? {:special_type    nil
                                                 :visibility_type :normal
                                                 :base_type       :type/Text}))
```

Comment:
why did this change from `false` to `true`? (Not saying it's wrong, just trying to understand the changes that are being made in this PR)